CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.17](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.17) - 2021-09-10
```

Comment:
CHANGELOG.md content looks great!

Changes to this file should be in a separate PR and we should remember to change the release date to the date when we actually tag the release.

Reply:
Thanks! Reverted changelog in 9b5172a, I'll create a separate PR